src/main/groovy/jenkins/automation/utils/GheUtils.groovy

```diff
@@ -0,0 +1,66 @@
+package jenkins.automation.utils
+
+/**
+ * Utility class to provide nicer, terser DSL for common tasks
+ */
+class GheUtils {
+
+/**
+ *
+ * Utility method to set a project repo as a GHE repo.
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#using-ghe-utility" target="_blank">example</a>
+ * @param context delegate
+ * @param gheProject A reference to the GHE project as 'username/reponame'
+ * @param gheHostname The GHE hostname, i.e. github.org.tld
+ */
+    static void gheRepo(context,
+                        String gheProject,
+                        String gheHostname,
+                        String gheRefspec = null,
+                        String gheBranch = null
+    ) {
+        context.with {
+            scm {
+                git {
+                    remote {
+                        github(gheProject, "https", gheHostname)
+                        if (gheRefspec) {
+                        refspec(gheRefspec)
+                        }
+                    }
+                    if (gheBranch) {
+                      branch(gheBranch)
+                    }
+                }
+            }
+        }
+    }
+/**
+ * Utility method to create a GitHub PR Builder trigger
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#using-ghe-utility" target="_blank">example</a>
+ * @param context delegate
+ * @param ghePrCron for use when ghePrHooks is false. Uses standard cron syntax
+ * @param ghePrHooks whether or not to install webhooks in the remote repo
+ * @param ghePrOrgList a string of organizations to whitelist for PRs
+ * @param gheAuthId the Jenkins credential ID to authenticate with
+ */
+    static void ghePrTrigger(context,
+                             ghePrCron = null,
+                             Boolean ghePrHooks = true,
+                             String ghePrOrgsList = 'jenkins',
+                             String gheAuthId
+    ) {
+        context.configure { node ->
+            node / 'triggers' / 'org.jenkinsci.plugins.ghprb.GhprbTrigger'( plugin: 'ghprb') {
+                spec(ghePrCron)
+                cron(ghePrCron)
+                configVersion('3')
```

Comment:
Magical number '3'?

Reply:
I got nothing on this one other than it seemed like a necessary part of the configure block.